src/core/lib/iomgr/error.c

```diff
@@ -249,6 +249,9 @@ grpc_error *grpc_error_create(const char *file, int line, const char *desc,
 
 static grpc_error *copy_error_and_unref(grpc_error *in) {
   GPR_TIMER_BEGIN("copy_error_and_unref", 0);
+  if (gpr_ref_is_unique(&in->refs)) {
```

Comment:
This should be moved down to line 269 (maybe as an `else if` on the main branch path): if `in` is special, then we'll segfault if we try to refer to `refs`.